tests/integration/src/lib.rs

```diff
@@ -31,3 +31,6 @@ mod assets;
 
 #[cfg(feature = "shibuya")]
 mod xvm;
+
+#[cfg(feature = "shibuya")]
```

Comment:
You can extend it to all runtimes I guess.

Reply:
yes, it should be extended, thanks.